generator/src/test/resources/generator/entity-relationship/extensions/Bar.pdl

```diff
@@ -0,0 +1,3 @@
+record Bar {
```

Comment:
Extension schema will not have such use case. It always includes base schema from /pegasus